[mHTTP GET /brewing 200 [1.28, 127.0.0.1:60803][0m
[36mHTTP GET /static/js/server_connection.js 304 [0.15, 127.0.0.1:60803][0m
WebSocket HANDSHAKING /ws/socket-server/ [127.0.0.1:60812]
WebSocket CONNECT /ws/socket-server/ [127.0.0.1:60812]
Sat Dec  3 14:49:23 2022: Another client connected
Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\Users\joni0\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\joni0\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\utils\deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "C:\Users\joni0\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
  File "C:\Users\joni0\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\http\request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\Users\joni0\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\joni0\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\utils\deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "C:\Users\joni0\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
  File "C:\Users\joni0\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\django\http\request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Bad Request: /favicon.ico
Not Found: /ws/socket-server/
Not Found: /ws/socket-server/
Not Found: /ws/socket-server/
Not Found: /ws/socket-server/
Not Found: /ws/socket-server/
Not Found: /ws/socket-server/
Not Found: /ws/socket-server/
C:\Users\joni0\OneDrive\Desktop\brew_seminar\brew_system\settings.py changed, reloading.
C:\Users\joni0\OneDrive\Desktop\brew_seminar\brew_system\settings.py changed, reloading.
Not Found: /ws/socket-server/
Not Found: /ws/socket-server/
Not Found: /ws/socket-server/
Not Found: /ws/socket-server/
